provider/src/main/java/com/cloudera/director/google/sql/GoogleCloudSQLInstanceTemplateConfigurationProperty.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.director.google.sql;
+
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.ADMIN_PASSWORD;
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.ADMIN_USERNAME;
+import static com.cloudera.director.spi.v1.database.DatabaseServerInstanceTemplate.DatabaseServerInstanceTemplateConfigurationPropertyToken.TYPE;
+
+import com.cloudera.director.spi.v1.model.ConfigurationProperty;
+import com.cloudera.director.spi.v1.model.ConfigurationPropertyToken;
+import com.cloudera.director.spi.v1.model.util.SimpleConfigurationPropertyBuilder;
+
+
+/**
+ * Google Cloud SQL instance template configuration properties.
+ */
+public enum GoogleCloudSQLInstanceTemplateConfigurationProperty implements ConfigurationPropertyToken {
+
+  TIER(new SimpleConfigurationPropertyBuilder()
+      .configKey("tier")
+      .name("Tier")
+      .defaultValue("D1")
+      .defaultDescription("The tier of your database. This affects performance and how much you will be charged.")
```

Comment:
Looks good. I can't remember where I originally copy/pasted this description from. I think I took it from Pantheon. Now it is different: 
"Your database tier affects performance and billing. Higher tiers offer your database more RAM, storage, concurrent connections, and I/O requests. After you select a tier, check the "Billing plan" section below to see your charges. For more details, see the Cloud SQL pricing guide."
Probably we should use this description to make it consistent with Pantheon (have to remove the last line though). What do you think?